docs/source/examples/unbounded.py

```diff
@@ -0,0 +1,20 @@
+from gpkit import Variable, Model
+from gpkit.tools import determine_unbounded_variables
+
+A = Variable("A")
+D = Variable("D")
+F = Variable("F")
+mi = Variable("m_i")
+
+Fs = Variable("Fs", 0.9)
+mb = Variable("m_b", 0.4)
+V = Variable("V", 300)
+
+m = Model(F, [F >= D + V**2,
+              D == V**2*A,
+              V >= mi + mb,
+              Fs <= mi,
+             ])
+# sol = m.solve("mosek")  # returns UNKNOWN
+bounds = determine_unbounded_variables(m, "mosek")
```

Comment:
@bqpd, note that removing `"mosek"` from the line above now causes the unit tests to fail. In a fun twist, it seems that cvxopt returns unknown on the problem with all variables bounded (even though it solved the original problem).